sdks/python/apache_beam/testing/test_utils.py

```diff
@@ -137,6 +137,12 @@ def delete_files(file_paths):
   FileSystems.delete(file_paths)
 
 
+def cleanup_snapshots(sub_client, snaps):
```

Comment:
I was using this in the `PubsubBasedCache` implementation (apache/beam#8961), but I figured that since these functions are in a *test_utils.py* file and I am not using them in a test I might as well just write my own.